tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRules.cs

```diff
@@ -135,8 +135,8 @@ public static Result IsAwsSqs(this MockSpan span) => Result.FromSpan(span)
                 .IsOptional("aws.region")
```

Comment:
The unified span naming tags also requires `messaging.system` on mesaaging spans. In this case, the value should be `AmazonSQS`